command/c/chown.md

```diff
@@ -0,0 +1,282 @@
+
+_Output:_
+```shell
+-rw-rw-r-- 1 sasa sasa 0 Dec  3 21:42 test.txt
+
+```
+
+Now, Suppose I want to change it to user `root` from user `sasa` then it can be done by below command: 
+
+```shell
+sudo chown root test.txt
+```
+
+_Output:_
+```shell
+-rw-rw-r-- 1 root sasa 0 Dec  3 21:42 test.txt
+
+```
+
+In above output section, ownership changed to user `root` from user `sasa`. Group `sasa` still has ownership. 
+
+_Note:_
+
+You can also use owner `user_id` in place of owner to change ownership in above example like: 
+    
+```shell
+id -u user_name
+```
+Suppose, output is 1000.
+
+Now, you can replace `user_name` with its `user_id`. 
+Example: In place of `sudo chown user_name test.txt`, you can use below:
+
+```shell
+sudo chown 1000 test.txt
+
+```
+
+_Another Note:_
+For changing ownership of folder, you can do it similarly like:
+
+```shell
+sudo chown root test_folder
+```
+It gives ownership of test_folder to `root` user.
+ 
+
+### 2. Changing the group of a file. 
+
+First checking the ownership of file from below command:
+
+```shell
+ls -l test.txt
+```
+Suppose output is:
+
+_Output:_
+```shell
+-rw-rw-r-- 1 sasa sasa 0 Dec  3 21:42 test.txt
+
+```
+
+Now, Suppose I want to change it to group `root` then it can be done by below command: 
+
+```shell
+sudo chown :root test.txt
+```
+
+_Output:_
+```shell
+-rw-rw-r-- 1 root root 0 Dec  3 21:42 test.txt
+
+```
+
+Changed group of file `test.txt` changed from `sasa` to `root`.
+
+
+_Note:_
+
+You can also use owner `group_id` in place of `group` to change ownership in above example like: 
+    
+```shell
+id -g group_name
+```
+Example: 
+
+```shell
+id -g root
+```
+Suppose, output is 0 .
+
+Now, you can replace `user_name` with its `user_id`. 
+Example: In place of "sudo chown :0 test.txt", you can use below:
+
+```shell
+sudo chown 1000 test.txt
+
+```
+
+### 3. Changing both file ownership and group at once. 
+
+First checking the owner ship of file from below command:
+
+```shell
+ls -l test.txt
+```
+Suppose output is:
+
+_Output:_
+```shell
+-rw-rw-r-- 1 root root 0 Dec  3 21:42 test.txt
+
+```
+
+Now, Suppose I want to change it to user `sasa` and group `sasa` then it can be done by below command: 
+
+```shell
+sudo chown sasa:sasa test.txt
+```
+
+_Output:_
+```shell
+-rw-rw-r-- 1 sasa sasa 0 Dec  3 21:42 test.txt
+
+```
+
+Ownership changed to user `sasa` and  group `sasa`.
+
+
+### 4. Changing owner of multiple files 
+
+You can use different files with space in between them to change ownership of multiple files at once. 
+
+```shell
+sudo chown owner_name file1 file2 file3 
+```
+### 5. Copying owner and group of file to another file. 
+
+Syntax: 
+```shell
+sudo chown --reference=source_file destination file
+```
+
+
+## Flags
+-   [`--reference=RFILE `](#referencerfile)
+-   [`-v, --verbose `](#v---verbose)
+-   [`-c, --changes `](#c---changes)
+-   [`-R, --recursive`](#r---recursive)
+-   [`--from=CURRENT_OWNER:CURRENT_GROUP`](#fromcurrent_ownercurrent_group)
```

Comment:
When I click on this it doesn't move to the link referenced

Reply:
solve this issue using html tag